json_diff_test.go

```diff
@@ -246,9 +244,12 @@ func TestShouldReturnFielsdWhenMoreThanOneFieldAreDifferent(t *testing.T) {
 		"num":   2,
 	}
 
+	expected := []string{"array", "json", "num", "str"}
 	actual := Diff(value, value2)
 
-	assert.Contains(t, actual, "json", "array", "str", "num")
+	if !reflect.DeepEqual(expected, actual) {
+		t.Error("Test failed. Expected", expected, "but returned", actual)
```

Comment:
Got curious if what will be printed on this case will be easy to read (instead of Errorf)

Reply:
Would be something like this:

--- FAIL: TestShouldReturnFielsdWhenMoreThanOneFieldIsDifferent (0.00s)
	json_diff_test.go:251: Test failed. Expected [array json num str] but returned [str num json array]

What do you think?

Reply:
Pretty cool, did not know it would format like that =D